inc/admin/metabox/manager.php

```diff
@@ -33,10 +43,17 @@ final class Manager {
 	 */
 	public function init() {
 		add_action( 'add_meta_boxes', array( $this, 'add' ) );
-		add_action( 'admin_init', array( $this, 'define_controls' ) );
-		add_action( 'admin_init', array( $this, 'load_controls' ) );
+		add_action( 'init', array( $this, 'define_controls' ) );
```

Comment:
if we need to load this at init hook, we need to make sure that they are not loading on frontend, i.e add an is_admin check maybe